content/blog/2022/03/2022-03-25-systemd-migration.adoc

```diff
@@ -0,0 +1,156 @@
+Users who are accustomed to editing an existing set of defaults must refer to the (read-only) service unit side-by-side when editing the drop-in unit
+or use a command like `systemctl edit jenkins --full`, which copies the original service unit instead of creating a drop-in unit.
+
+A final point to mention about the service unit is its use of specifiers,
+which may be unfamiliar to some users.
+The `systemd(1)` documentation contains link:https://www.freedesktop.org/software/systemd/man/systemd.unit.html#id-1.13.3[a table of specifiers available in unit files].
+
+==== Migration
+
+The Jenkins project ships logic to automatically migrate the System V `init(8)` configuration file to the new `systemd(1)` `override.conf` format.
+This migration logic does nothing if an `override.conf` file already exists,
+which would be an indication that the migration script already ran
+or that the user has made their own customizations that should be preserved.
+If `override.conf` does not exist, package installation migrates the old System V `init(8)` configuration file to `override.conf`.
+
+==== Logging
+
+The `systemd(1)` package also uses `systemd-journald(8)` for logging by default.
+Rather than creating a log file in `/var/log/jenkins.log`,
+Jenkins now logs to the system's journal.
+Log entries may be viewed with `journalctl -u jenkins`.
+This is perhaps the most noticeable user-visible change in this migration.
+
+==== Startup notifications
+
+The System V `init(8)` logic was asynchronous; i.e., running `/etc/init.d/jenkins start` would return prior to the completion of Jenkins startup.
+The `systemd(1)` logic is synchronous; i.e., running `systemctl start jenkins` will block until Jenkins signals successful startup.
+This allows system administrators to write automation to programmatically deploy Jenkins using modern tools like Ansible.
+
+=== Reporting issues
+
+If you find a regression, please file a bug report in link:https://issues.jenkins.io/[Jira].
+When reporting an issue, include the following information:
+
+. Use the `core` component.
+. Provide the name, version, and architecture of the Linux distribution you are using (e.g., Ubuntu 20.04.4 LTS x86_64).
+. Provide the contents of the old System V `init(8)` configuration in `/etc/{default,sysconfig}/jenkins`, sanitized as necessary.
+. Provide the contents of the `systemd(1)` drop-in unit in `/etc/systemd/system/jenkins.service.d/override.conf`, sanitized as necessary.
+. Provide steps to reproduce the issue from _scratch_ on a minimal Jenkins installation; the scenario should fail when the steps are followed on Jenkins 2.335 or later and pass when the steps are followed on Jenkins 2.334 or earlier.
+
+=== Conclusion
+
+We expect to see a bit of disruption from these changes but hope that in the long run they will save time for core and plugin developers and lead to a more secure and stable tool. Please reach out on the developers' list with any questions or suggestions.
```

Comment:
Optional that sentence per line is easier to review and recommended in the style guide.
```suggestion
We expect to see a bit of disruption from these changes but hope that in the long run they will save time for core and plugin developers and lead to a more secure and stable tool.
Please reach out on the link:https://groups.google.com/g/jenkinsci-dev[developers' mailing list] with any questions or suggestions.
```